mdx/granule_metadata_extractor/processing/process_sbuparsimpacts.py

```diff
@@ -41,15 +41,33 @@ def get_variables_min_max(self, nc, file_path):
         :param file_path: file path
         :return:
         """
-        if '_RT.nc' in file_path:
-            minTime, maxTime, minlat, maxlat, minlon, maxlon = self.get_RT_metadata(nc)
-        elif '_MAN.nc' in file_path:
-            minTime, maxTime, minlat, maxlat, minlon, maxlon = self.get_MAN_SB_metadata(nc)
-        elif '_SB.nc' in file_path:
-            minTime, maxTime, minlat, maxlat, minlon, maxlon = self.get_MAN_SB_metadata(nc)
+        if 'parsivel_2022' in file_path:
+            minTime, maxTime, minlat, maxlat, minlon, maxlon = self.get_2022_metadata(nc)
+        else: # 2020 data files

```

Comment:
Rather than doing:
```
else:
    if:
    ...
    elif:
    ...
```
you can just use
```
elif:
...
elif:
...
```
in the future. This will help avoid nesting if not necessary and will serve the same purpose.